app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(line: str) -> None:
+    old_file_name = line.split()[1]
+    new_file_name = line.split()[2]
```

Comment:
The variable names 'old_file_name' and 'new_file_name' are descriptive and correct, which is good practice.